### DISCUSSION TITLE
main branch's train_vits_tts.py won't work, you need to switch to dev branch

### DISCUSSION THREAD

**Discussion by makerjackie** [coqui-ai/TTS#1464] [General]
just to mention that .... It wasted me a little time...

https://github.com/coqui-ai/TTS/blob/dev/recipes/multilingual/vits_tts/train_vits_tts.py

**Comment by erogol** [comment]
what was the problem?

**Reply by makerjackie** [reply]
Some api has been change, like `CharactersConfig`

**Reply by erogol** [reply]
Yeah makes sense. As we've not released a major version yet, such API conflicts is yet to be expected :(
